docs/source/snowpark/index.rst

```diff
@@ -0,0 +1,33 @@
+Snowpark API Reference (Python)
```

Comment:
what title should we put here? @sfc-gh-mabrennan

Reply:
What is wrong with the current title? I don't understand the context of your question.

Reply:
Do you want to change it to Snowpark Python API Reference? If so, that's fine with me.

Reply:
top level title -  Snowpark API Reference (Python). Secondary level 3 expandable sections:
Snowpark Session
Snowpark API
Snowpark pandas API